pandas/tests/frame/test_analytics.py

```diff
@@ -2073,3 +2073,16 @@ def test_n_duplicate_index(self, df_duplicates, n, order):
         result = df.nlargest(n, order)
         expected = df.sort_values(order, ascending=False).head(n)
         tm.assert_frame_equal(result, expected)
+
+    def test_series_broadcasting(self):
+        # smoke test for numpy warnings
+        # GH 16378, GH 16306
+        df = DataFrame({'A': [np.nan, 2.0, np.nan]})
+        s = Series([1, 1, 1])
+
+        with tm.assert_produces_warning(None):
+            df.clip_lower(s, axis=0)
+            df.gt(s, axis=0)
```

Comment:
``pd.DataFrame([0,0]).gt(pd.Series([0, np.nan]), axis=0)``

can you also add this test (the Series has the nan)

maybe just loop over these e.g.

```
for op in ['lt', 'le', 'ge', 'lt', 'eq', 'ne']:
     getattr(df, op)(s, axis=0)
```